routing_packager_app/config.py

```diff
@@ -79,6 +79,19 @@ def get_tmp_data_dir(self) -> Path:
 
         return tmp_data_dir
 
+    def get_logging_dir(self) -> Path:
```

Comment:
Internally all logs are written into the `tmp_data_dir` logs subdirectory, so they are easily shared with the FastAPI app.